### PR TITLE
Feature/download ascii

### LINKED ISSUE #32: Download Option for Final Ascii Output
Create user interface and logic to download final Ascii art output

### PR DESCRIPTION
**Pull request by elizasohn**
Creates a "download" button that allows users to download and save the ascii art produced.
Uses packages "html2canvas" and "downloadjs"

### CODE REVIEW

File: artscii/src/services/downloadAscii.js
Comment (srimel) [inline artscii/src/services/downloadAscii.js:1]: Looks good and works as advertised! Can you also include the dependency changes in your package.json for this PR?

**Review by srimel — CHANGES_REQUESTED** [review]

File: artscii/src/services/downloadAscii.js
Comment (unfinishedideas) [reply artscii/src/services/downloadAscii.js:1]: Yeah, I wonder why it's not pushing those changes to the package.json?

Comment (srimel) [reply artscii/src/services/downloadAscii.js:1]: Updated the PR

**Review by srimel — APPROVED** [review]

**Review by unfinishedideas — APPROVED** [review]
Looking good!

**Pull request merged**